src/plugin.js

```diff
@@ -36,6 +36,15 @@ export const hasSession = (sessions, initData) => {
   return false;
 };
 
+export const removeSession = (sessions, initData) => {
+  for (let i = 0; i < sessions.length; i++) {
```

Comment:
I think we could use a filter here if we wanted to (unclear as to whether `sessions` is a `const`), something like:

`sessions = sessions.filter(session => session.initData !== initData)`;

Reply:
That should create a new array and change the local reference, but not modify the actual array that we want to modify.

Reply:
Yeah, I guess we never re-set the sessions here, just modify array.  Bummer.